{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40790979",
  "multipleFaultNumbers": null,
  "date": "2020-12-18T15:40:56",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-18T15:40:56",
  "information": "",
  "actualTimeOfRestoration": "2020-12-18T15:40:56",
  "outageCentrePoint": {
    "lat": 53.591865,
    "lng": -2.459572
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL1 5RS, BL1 5RT, BL1 5RW, BL1 5RX, BL1 6EP, BL1 6ER, BL1 6ES, BL1 6ET, BL1 6EU, BL1 6EW, BL1 6HH, BL1 6HJ",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GS"
}